api/controller/common.proto

```diff
@@ -8,10 +8,12 @@ import "github.com/mwitkow/go-proto-validators/validator.proto";
 
 // RunningOperation respresents a long-running operation.
 message RunningOperation {
-  // Progress from 0.0 to 1.0; can decrease compared to the previous value.
-  float progress = 1;
+  // Finished steps of the operaion.
```

Comment:
Can it decrease compared to the previous value?

Reply:
Yes, it's possible. @jprukner , please update comment.

Reply:
Comment updated.